.env.sample

```diff
@@ -20,7 +20,7 @@ IDP_CLIENT_SECRET=''
 IDP_GROUP_OTOMI_ADMIN='e69ded30-0882-4490-8e0f-2e67625a0693'
 IDP_GROUP_TEAM_ADMIN='3c63814c-59df-46c3-9a69-d9e1c3611097'
 IDP_GROUP_MAPPINGS_TEAMS='{"team-otomi":"28010af7-9535-4265-8689-50f51f8f2c87"}'
-KEYCLOAK_ADDRESS='https://keycloak.dev.gke.otomi.cloud'
+KC_HOSTNAME='https://keycloak.dev.gke.otomi.cloud'
```

Comment:
if it is a hostname, then why in the example there is an URL ?

Reply:
updated it to [KC_HOSTNAME_URL](https://www.keycloak.org/server/all-config?q=hostname#category-hostname)